app/src/main/java/com/iic/mokojin/activities/PlayerQueueFragment.java

```diff
@@ -101,15 +101,17 @@ public void run() {
                     scheduleUpdateClock();
                     return;
                 }
-                getActivity().runOnUiThread(new Runnable() {
-                    @Override
-                    public void run() {
-                        if (null != mQueueAdapter) {
-                            mQueueAdapter.notifyDataSetChanged();
-                            scheduleUpdateClock();
+                if (null != getActivity()) {
```

Comment:
Apparently this can be called when opening the app through the push notification for invited players. In this case, the activity isn't fully loaded and GetActivity() is null